subprojects/docs/src/snippets/developingPlugins/testingPlugins/kotlin/url-verifier-plugin/build.gradle.kts

```diff
@@ -58,7 +58,7 @@ dependencies {
     "functionalTestImplementation"("org.spockframework:spock-core")
 }
 
-tasks.withType<Test>().configureEach {
+tasks.withType<Test> {
```

Comment:
This is a very subtle point, but calling the 2-arg `withType` method that takes an `Action` actually *is* eager.  Calling the 1-arg `withType` and then calling `configureEach` on the result is lazy.  See: https://docs.gradle.org/current/userguide/task_configuration_avoidance.html#sec:how_do_i_defer_configuration and the DSL on https://docs.gradle.org/current/javadoc/org/gradle/api/DomainObjectCollection.html

I should have been more explicit about this in my initial review, sorry.  These should be updated to avoid confusing others with this usage, there are a number of places in this PR to update.